src/protocols/ethereum/views/AccountDetails.vue

```diff
@@ -32,10 +41,24 @@ export default defineComponent({
     IonContent,
     AccountDetailsNavigation,
   },
-  setup() {
+  props: {
+    ionicLifecycleStatus: { type: String as PropType<IonicLifecycleStatus>, default: null },
```

Comment:
What do you think about replacing this `ionicLifecycleStatus` with a bool prop? Is this possible? I see we have 4 possible values for this, but isn't 2 of them truthy and 2 falsy?

Or maybe having 2 separate bool props: `pageDidEnter` and `pageWillEnter`.

The benefit of boolean prop is that we can avoid importing and using the `IonicLifecycleStatus` type. Also we could stop using the `'didEnter'` string values. Instead the prop could be used directly in the template to display or hide the elements.